src/en/charms-storage.md

```diff
@@ -133,6 +135,10 @@ rootfs   rootfs
 tmpfs    tmpfs     
 ```
 
+!!! Note:
+    The name given to a default storage pool will often be the same as the
+    name of the storage pool upon which it is based.
```

Comment:
I think you mean "same as the name of the storage provider upon which it is based"? in this case the default pool is "ebs", which  has the same name as the provider that services that pool